lib/duplicate-slashes.js

```diff
@@ -0,0 +1,37 @@
+function removeDuplicateSlashes (path) {
```

Comment:
Can you remove this, since you discovered that the regexp version is working faster?